src/netlink_namespace.c

```diff
@@ -116,7 +116,8 @@ static const char *config_keys[] = {"Interface", "VerboseInterface",
 static int config_keys_num = STATIC_ARRAY_SIZE(config_keys);
 
 /* Will store namespace name here */
-static char **ns_name = NULL;
+static char *ns_name = NULL;
+static struct mnl_socket *nloriginal, *nltarget;
```

Comment:
Do you need to keep a netlink socket to the original network namespace ? - What's it used for ?